.github/workflows/lint.yml

```diff
@@ -131,7 +131,7 @@ jobs:
         run: echo "::add-matcher::.github/sphinx-problem-matcher.json"
 
       - name: Build docs
-        run: |
+        run: |-
```

Comment:
What's this do?

Reply:
This was accidental actually. See 
![image](https://github.com/user-attachments/assets/e0c04d71-a1b5-4cf4-b966-9f5452e97382)